pkg/apis/rook.io/v1alpha1/storage.go

```diff
@@ -15,20 +15,18 @@ limitations under the License.
 */
 
 // Package osd for the Ceph OSDs.
-package osd
+package v1alpha1
 
-import (
-	cephosd "github.com/rook/rook/pkg/ceph/osd"
-)
+const bluestore = "bluestore"
```

Comment:
i would still name this `defaultStore` just to explicitely convey that bluestore if our default. Also if we change to a different store like filestore, we just have to modify this const.

Reply:
we had to decouple this from the package that defined the default store, so it actually seemed better to define bluestore explicitly instead of defining the default in another place.